config/cyan/config.clj

```diff
@@ -0,0 +1,19 @@
+{:output-dir "/apps/data/marcgrep-cyan"
+
+ :state-file "/apps/data/marcgrep-cyan/marcgrep.dat"
+
+ ;; Note: the total number of active threads will be (roughly) these two values multiplied.
+ :worker-threads 4
+ :max-concurrent-jobs 2
+
+ :marc-destination-list [marcgrep.destinations.marcfile marcgrep.destinations.plaintext]
+
+ :marc-source-list [{:description "Bibliographic and Holdings data"
+                     :driver marcgrep.sources.marc-file
+                     :marc-files ["/apps/data/folio-cache/MARC_BIB.mrc"]}
```

Comment:
Is this an mfhd file out of interest?

Reply:
It's an .mrc file generated from the combined bibliographic + holdings records stored in the folio-cache. I've asked Julie Watson and her team to check that the combined format works for them.